includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -215,6 +225,37 @@ public function register_routes() {
 		);
 	}
 
+	/**
+	 * Validate the regular statement descriptor.
+	 *
+	 * @param mixed           $value The value being validated.
```

Comment:
Can the `$value` param be something other than `string`?

Reply:
Good catch. Updated it in ce8b9e3.